draft-packages/modal/package.json

```diff
@@ -25,7 +25,7 @@
   "private": false,
   "license": "MIT",
   "dependencies": {
-    "@headlessui/react": "<=1.5.0",
+    "@headlessui/react": "^1.7.15",
```

Comment:
as per note in https://github.com/cultureamp/kaizen-design-system/pull/2734, https://github.com/tailwindlabs/headlessui/issues/1417 is resolved